agent/api/task/taskvolume.go

```diff
@@ -51,7 +52,8 @@ func (tv *TaskVolume) UnmarshalJSON(b []byte) error {
 
 	volumeType, ok := intermediate["type"]
 	if !ok {
-		return errors.New("invalid Volume: must include a type")
+		volumeType = []byte(`"host"`)
+		seelog.Infof("Unmarshal task volume: volume type not specified, default to host")
```

Comment:
This can just be at the debug level.

Reply:
I think it should be `Info` level, as it's translating the task resource in a specific case.